src/diet.ts

```diff
@@ -84,6 +84,7 @@ import { Potion, PotionTier } from "./potions";
 import { garboValue } from "./session";
 import synthesize from "./synthesis";
 import { estimatedGarboTurns } from "./turns";
+import { BooleanProperty } from "libram/dist/propertyTypes";
```

Comment:
in libram 8.1 you should be able to import this directly; be sure to `yarn install` if you can't